spec/react_on_rails/configuration_spec.rb

```diff
@@ -13,6 +13,8 @@ module ReactOnRails
 
     before do
       allow(ReactOnRails::PackerUtils).to receive(:using_packer?).and_return(using_packer)
+      allow(ReactOnRails::Utils).to receive(:gem_available?).and_call_original
+      allow(ReactOnRails::Utils).to receive(:gem_available?).with("webpacker").and_return(false)
```

Comment:
Probably can remove this once you remove webpacker from the `spec/dummy/Gemfile.lock`